alab_management/experiment_manager.py

```diff
@@ -158,14 +156,13 @@ def mark_completed_experiments(self):
 
             # if all the tasks of an experiment have been finished
             if all(
-                self.task_view.get_status(task_id=task_id)
-                in {
-                    TaskStatus.COMPLETED,
-                    TaskStatus.ERROR,
-                    TaskStatus.CANCELLED,
-                    TaskStatus.STOPPED,
```

Comment:
why do we want to remove the stopped?

Reply:
Because it is not used anywhere in the current alabos repo. I deleted the TaskStatus.STOPPED enum.